tracker/event_tracker_store.go

```diff
@@ -25,15 +29,15 @@ var (
 type EventTrackerStore struct {
 	conn          *bolt.DB
 	finalityDepth uint64
-	notifierCh    chan<- []*ethgo.Log
+	subscriber    eventSubscription
 	logger        hcf.Logger
 }
 
 // NewEventTrackerStore creates a new EventTrackerStore
 func NewEventTrackerStore(
 	path string,
 	finalityDepth uint64,
```

Comment:
Can we call this `numBlockConfirmations`? That is how it is usually referred to this value.

Reply:
renamed https://github.com/0xPolygon/polygon-edge/pull/1228/commits/2fe1d5c1fba88ff426e9a55dbbccf1e044921ad2